pkg/console/subresource/route/route.go

```diff
@@ -162,31 +170,41 @@ func wildcard() routev1.WildcardPolicyType {
 	return routev1.WildcardPolicyNone
 }
 
+func GetCanonicalHost(route *routev1.Route) (string, error) {
```

Comment:
Log spam warning: if you're not caching this result and call this on every sync, consider logging throughout at a different verbosity level

Reply:
Thanks!  Fortunately, our sync loop idles out & is pretty quiet after the initial flurry, so we haven't worried about log levels yet.